rasterio/enums.py

```diff
@@ -104,3 +104,32 @@ class MergeAlg(Enum):
     """Available rasterization algorithms"""
     replace = 'REPLACE'
     add = 'ADD'
+
+
+class WktVersion(Enum):
```

Comment:
In PROJ 7 the enum is named PJ_WKT_TYPE. What would you think about changing the enum name to WktType?

Reply:
It is currently designed to be consistent with pyproj: https://pyproj4.github.io/pyproj/stable/api/enums.html#pyproj.enums.WktVersion.

Reply:
GDAL calls it `FORMAT`. I guess there isn't much consistency 😂.

Reply:
My preference is to keep it consistent with pyproj. But, in the end I guess it doesn't really matter. If you prefer WktType, we can do that. If we go that direction, then the input kwarg should be renamed to `type`. Actually, that is a python reserved name ... so `wkt_type`.

Reply:
To be consistent with PROJ, GDAL, and pyproj perhaps we should call it `variant` :joy:

Reply:
I agree, let's make pyproj and rasterio consistent.